### ISSUE TITLE
Probabilistic PCA with latent dim > 1 yields incorrect estimates

### ISSUE THREAD

**Issue by andrewcharlesjones** [tensorflow/probability#1128]
Hi all, in the probabilistic PCA tutorial, there seems to be an issue when the latent dimension k is set such that k>1. Specifically, the MAP estimates of Z and W appear to be constrained so that all latent variables are identical (i.e., the columns of W and the rows of Z are identical).

This can be recreated by running the [code in the PPCA tutorial](https://github.com/tensorflow/probability/blob/master/tensorflow_probability/examples/jupyter_notebooks/Probabilistic_PCA.ipynb), but setting `latent_dim=3`, for example.

Any help would be appreciated in resolving this. Not sure whether it might be an issue with the batching, initialization, or something else.

Best,
Andy

**Comment by davmre** [comment]
I bet this is due to initialization: it looks like the variables are initialized with identical values (`tf.ones`) and without any randomness in the optimization there's nothing to break the symmetry. What happens if you initialize to random values?

```python
w = tf.Variable(tf.random.normal([data_dim, latent_dim]))
z = tf.Variable(tf.random.normal([latent_dim, num_datapoints]))

target_log_prob_fn = lambda w, z: model.log_prob((w, z, x_train))
losses = tfp.math.minimize(
    lambda: -target_log_prob_fn(w, z),
    optimizer=tf.optimizers.Adam(learning_rate=0.05),
    num_steps=200)
```

**Comment by andrewcharlesjones** [comment]
Yup, that seems to be it. Thanks Dave!

Perhaps the initialization should be changed in the tutorial? (The initialization should be randomized for the variational distributions too I think).

-Andy
